recipe/meta.yaml

```diff
@@ -13,26 +13,30 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 202
-  # NOTE: Conda forge lacks openblas on Windows, so skip the Windows build.
-  skip: true  # [win]
-  features:
-    - blas_{{ variant }}
```

Comment:
Keep this feature. If you remove this, older builds will be given priority if `blas=*=openblas` package is installed.